data/json/items/armor.json

```diff
@@ -10170,5 +10170,124 @@
     "material_thickness": 1,
     "environmental_protection": 1,
     "flags": [ "OVERSIZE", "OUTER", "WATERPROOF", "ALLOWS_NATURAL_ATTACKS" ]
-  }
+  },
+  {
+    "id": "the bigpack",
```

Comment:
No spaces in id names, that's a big nono.

Reply:
Thanks for the heads up